### PR TITLE
Eng/pr 6 - Game Menu (post rebase)

### LINKED ISSUE #6: Create Game Menu UI

### LINKED ISSUE #22: Rebase feature/baseMainMenu

### PR DESCRIPTION
**Pull request by Kerowyn202**

### CODE REVIEW

File: Modules/game_menu_ui.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# File to store data to create the game menu(s)
+import pygame
+import pygame_menu as pm
+
```
Comment (henslk1) [inline Modules/game_menu_ui.py:4]: Needs two new lines between import statements and classes.

File: Modules/game_menu_ui.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        self.home_menu.add.button(title="Settings")
+
+        # High Score button - when clicked, goes to High Scores
+        self.home_menu.add.button(title="High Score")
+
+        # Play button - when clicked, begins game
+        self.home_menu.add.button(title="Play")
+
+        # Quit button - exits game to desktop
+        self.home_menu.add.button(title="Quit")
```
Comment (henslk1) [inline Modules/game_menu_ui.py]: Needs one new line at the end of the file.

Comment (Kerowyn202) [reply Modules/game_menu_ui.py]: Fixed!

File: Modules/game_menu_ui.py (hunk 1 above)
Comment (Kerowyn202) [reply Modules/game_menu_ui.py:4]: fixed!

**Review by henslk1 — APPROVED** [review]
New lines resolved, looks great.

**Pull request merged**